src/utils/__tests__/navigation-actions.spec.js

```diff
@@ -118,6 +119,27 @@ describe('navigation actions', () => {
         expect(senseNavigation.goToSheet).toHaveBeenCalledWith('id3');
       });
     });
+
+    describe('odag navigation popup', () => {
+      it('should not call openOdagPopup when openOdagPopup is undefined', async () => {
+        const navigationObject = navigationActions.find((navigation) => navigation.value === 'odagLink');
```

Comment:
I think you would only need to do this once and use it in al three test cases

Reply:
Done.